adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/controllers/pages/AdyenComponentController.java

```diff
@@ -82,6 +83,14 @@ public String componentPayment(final Model model,
                                 final BindingResult bindingResult) throws AdyenComponentException {
         LOGGER.debug("Component paymentForm: " + adyenPaymentForm);
 
+        if(!isValidateSessionCart())
+        {
+            if (adyenPaymentForm.getBillingAddress() != null) {
+                adyenPaymentForm.resetFormExceptBillingAddress();
+            }
+            throw new AdyenComponentException("checkout.error.paymentethod.formentry.invalid");
```

Comment:
I see it's in multiple places, not really a priority to fix since I think it works anyway
```suggestion
            throw new AdyenComponentException("checkout.error.paymentmethod.formentry.invalid");
```

Reply:
This is typo in hybris.